# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ndiswrapper/no-depmod.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

depmod is verboten.

  - Rene Rebe <rene@exactcode.de>

--- ndis/driver/Makefile.vanilla	2005-09-07 20:22:15.000000000 +0200
+++ ndis/driver/Makefile	2005-09-07 20:22:22.000000000 +0200
@@ -223,7 +223,6 @@
 install: prereq_check x86_64_stubs gen_exports default stack_check
 	mkdir -p $(INST_DIR)
 	install -m 0644 $(MODULE) $(INST_DIR)
-	-/sbin/depmod -a
 
 dist:
 	@for file in $(DISTFILES); do \
